### PR TITLE
Arcade Driving

### PR DESCRIPTION
**Pull request by Zinka010**
- Created command-based driving of the robot
 - Implemented only arcade drive to allow new students to learn by adding more

**Testing performed**
 - Mergio

### CODE REVIEW

**Review by nrgill28 — APPROVED** [review]

File: src/main/java/frc/robot/commands/ExampleCommand.java [hunk 1]
```diff
@@ -7,26 +7,31 @@
...
    * Creates a new ExampleCommand.
    *
    * @param subsystem The subsystem used by this command.
    */
-  public ExampleCommand(ExampleSubsystem subsystem) {
-    m_subsystem = subsystem;
-    // Use addRequirements() here to declare subsystem dependencies.
-    addRequirements(subsystem);
-  }
+  // public ExampleCommand(ExampleSubsystem subsystem) {
```
Comment (georgetzavelas) [inline src/main/java/frc/robot/commands/ExampleCommand.java]: Maybe we should prune this if it is not needed?

File: src/main/java/frc/robot/config/Config.java [hunk 2]
```diff
@@ -37,12 +37,38 @@
...
+    public static int RIGHT_CONTROL_STICK_Y = 5;
+    public static int RIGHT_CONTROL_STICK_X = 4;
+
+    public static boolean INVERT_FORWARD = robotSpecific(true, true, true);
+    public static boolean INVERT_SIDE= robotSpecific(false, false, false);
+
+
+    // Timeouts for sending CAN bus commands
+    public static final int
+            CAN_SHORT = 10,
```
Comment (georgetzavelas) [inline src/main/java/frc/robot/config/Config.java]: Would be a little clearer to use the same style and have explicit declarations on one line. Also having TIMEOUT in the name would clarify this when it is used in other places in the codebase

File: src/main/java/frc/robot/subsystems/DriveBase.java [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+        leftRearTalon = new WPI_TalonSRX(Config.LEFT_REAR_TALON);
+        rightFrontTalon = new WPI_TalonSRX(Config.RIGHT_FRONT_TALON);
+        rightRearTalon = new WPI_TalonSRX(Config.RIGHT_REAR_TALON);
+
+        robotDriveBase = new DifferentialDrive(leftFrontTalon, rightFrontTalon);
+
+    }
+
+    public static void init() {
+        if (currentInstance == null) {
```
Comment (georgetzavelas) [inline src/main/java/frc/robot/subsystems/DriveBase.java:50]: Nice Singleton pattern use!

**Review by georgetzavelas — APPROVED** [review]
Looks good, some comments for your consideration

**Comment by Zinka010** [comment]
Addressed issues raised by @georgetzavelas

**Pull request merged**